hnn_core/network.py

```diff
@@ -518,3 +522,76 @@ def plot_spikes(self, ax=None, show=True):
         if show:
             plt.show()
         return ax.get_figure()
+
+    def write_spikes(self, fname, trial_idx=None):
+        """Write spike times to a file.
+
+        Parameters
+        ----------
+        fname : str
+            Full path to the output file (.txt)
+        trial_idx : list of int
+            Indices of selected trials. If None,
+            all trials are selected.
+
+        Outputs
+        -------
+        txt file at fname where rows correspond to
+            spikes and columns, delimited by '\\t',
+            correspond to 1) spike time (s),
+            2) spike gid, and 3) gid type
+        """
+        if trial_idx is None:
+            trial_idx = range(len(self.spiketimes))
+
+        spiketimes = []
+        spikegids = []
+        for idx in trial_idx:
+            spiketimes += self.spiketimes[idx]
+            spikegids += self.spikegids[idx]
+
+        gidtypes = np.empty_like(spikegids, dtype='<U36')
+        for spike_type, gid_range in self.gid_dict.items():
+            gidtypes[np.in1d(spikegids, gid_range)] = spike_type
+
+        with open(fname, 'w') as f:
+            for spk_idx in range(len(spiketimes)):
+                f.write('{:.3f}\t{}\t{}\n'.format(spiketimes[spk_idx],
+                        int(spikegids[spk_idx]), gidtypes[spk_idx]))
+
+    def read_spikes(self, fname, append_trial=True):
```

Comment:
```suggestion
    def read_spikes(self, fname):
```

just allow `fname` to have a wildcard expression and loop over all the files that match that expression and return a list.